### DISCUSSION TITLE
[Feature Request] Simple back and forward navigation

### DISCUSSION THREAD

**Discussion by neverxxever** [standardnotes/forum#2622] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes and I agree with it.

**Is your feature request related to a problem? Please describe.**
I am always frustrated when I need to switch between two or three notes that are miles apart because I spend so much time scrolling and looking for the right ones.

**Describe the solution you'd like**
I think it would be very useful if we had simple back and forward buttons for navigating like a web browser. Ideally these would also work with integrated back/forward mouse buttons but I would be ecstatic to see the feature in any form. This feature is super useful in file explorers too.

**Describe alternatives you've considered**
If we were able to open more than one note at a time that would probably work but that's not really a feature I want and I think it's kind of contrary to the way Standard Notes works.
I could temporarily create a new note in the same folder or keep copies of relevant information in both notes (which I often do as a workaround) but this is messy and requires a lot of loose ends or tidying up.

**Additional context**
Love the app Mo, plan on using it forever whether you add this feature or not.

**Comment by JaspalSuri** [comment]
Hi, thanks for the feedback! I haven't forwarded it over yet for review, because I'd like to hear more from you first. 

May I ask why pinning the notes wouldn't help? 

Also, this is sort of along the lines of what you've mentioned, but you could add the same tag to all of the relevant/related notes and then click on the tag in the leftmost sidebar/pane to view all the notes with just that tag. 

One more thing that you could try is use [dynamic tags](https://standardnotes.org/help/42/how-do-i-view-a-list-of-untagged-notes-and-create-other-dynamic-filters) to pull up multiple relevant notes with different tags. 

I apologize if I'm overlooking something simple yet significant. 

Cheers!

**Comment by atmoio** [comment]
> If we were able to open more than one note at a time

This is something we're exploring for v4. Nothing concrete as of yet but I do hear you on the fact that multi-note navigation could use dedicated solutions.

**Comment by arielsvg** [comment]
Supporting the browser's back/forward button sounds like something we ought to do at some point. 🤔

**Comment by munkmode** [comment]
+1, I want to see simple back and forward navigation too. Evernote has this feature and it's useful.
